tests/env/node/search_example.js

```diff
@@ -7,14 +7,19 @@ const config = {
 }
 
 const client = new MeiliSearch(config)
-const uid = 'movies'
+const indexUid = 'movies'
 
 const addDataset = async () => {
-  const index = await client.getOrCreateIndex(uid)
+  await client.deleteIndex(indexUid)
+  const { uid } = await client.createIndex(indexUid)
+  await index.waitForTask(uid)
```

Comment:
I think it should be:
```suggestion
  await client.waitForTask(uid)
```
Because the index variable has not been instantiated yet.